packages/components/src/validated-form-controls/components/array-control.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * WordPress dependencies
+ */
+import { forwardRef, useRef } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { ControlWithError } from '../control-with-error';
```

Comment:
I noticed some Sass variable handling is wrong in the stylesheet, so I'll fix it in #71391. With that in mind though, I think we'll need some error styles for `FormTokenField` as well, similar to what we have for `ComboboxControl`.

https://github.com/WordPress/gutenberg/blob/775f0cea9ea3b0a5ea8a39982475f82a9a6ddae0/packages/components/src/validated-form-controls/style.scss#L16-L21